### PR TITLE
Edited variable name for duplicate check

### PR DESCRIPTION
**Pull request by Sheikh-Umar**

### CODE REVIEW

**Comment by davidten** [comment]
why?

**Comment by Sheikh-Umar** [comment]
Changed the name of the person object to so that it's in line with good naming convention.

File: docs/team/johndoe.adoc [hunk 1]
```diff
@@ -61,9 +61,6 @@ include::../UserGuide.adoc[tag=dataencryption]
 
 include::../DeveloperGuide.adoc[tag=undoredo]
 
-include::../DeveloperGuide.adoc[tag=dataencryption]
```
Comment (davidten) [inline docs/team/johndoe.adoc]: Why are you removing this as part of this PR?

Comment (Sheikh-Umar) [reply docs/team/johndoe.adoc]: I thought I made this change in another branch. I've added it back in.

**Comment by Sheikh-Umar** [comment]
I've added the dataencryption tag back to johndoe.adoc. You may review my PR and approve it.

**Pull request merged**